From ea1ce3b19cbb0308d2d5b45092ba223b9a7339fc Mon Sep 17 00:00:00 2001
From: Mel Gorman <mgorman@suse.de>
Date: Wed, 25 Jun 2014 10:07:24 +0100
Subject: [PATCH] mm: Rearrange zone fields into read-only, page alloc,
 statistics and page reclaim lines -post

References: VM Performance
Patch-mainline: No (Never)

This is a post-preparation patch to make a later patch easier to backport
from upstream.

Signed-off-by: Mel Gorman <mgorman@suse.de>
---
 include/linux/mmzone.h | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/include/linux/mmzone.h b/include/linux/mmzone.h
index 5c4149d..7faf1ba 100644
--- a/include/linux/mmzone.h
+++ b/include/linux/mmzone.h
@@ -481,9 +481,6 @@ struct zone {
 	spinlock_t		lru_lock;
 	struct lruvec		lruvec;
 
-	/* Evictions & activations on the inactive file list */
-	atomic_long_t		inactive_age;
-
 	unsigned long		pages_scanned;	   /* since last reclaim */
 
 #if defined CONFIG_COMPACTION || defined CONFIG_CMA
